pyproject.toml

```diff
@@ -56,7 +56,7 @@ typing-extensions = ">=4.8.0"
 encord = ">=0.1.159"
 numpy = ">=1.26.4"
 opencv-python = ">=4.1"
-
+PyMuPdf = ">1.25.0"
```

Comment:
This is a big no no IMO. I agree with your comment that this should be made optional.
So you do `pip install "encord-agents[pdf]" if you need it.
`encord-agents` has to be light weight in terms of dependencies. Cause what happens when we need to support DICOM or some exotic geospatial data type.